mythril/analysis/modules/delegatecall.py

```diff
@@ -18,7 +18,8 @@ def __init__(self):
         super().__init__(
             name="DELEGATECALL Usage in Fallback Function",
             swc_id=DELEGATECALL_TO_UNTRUSTED_CONTRACT,
-            hooks=["DELEGATECALL"],
+            pre_hooks=["DELEGATECALL"],
+            post_hooks=[],
```

Comment:
I think you can just default post_hooks to [] in the constructor itself rather than adding it in each place, now it doesn't matter anyway.

Reply:
I'll change that and do the same with `pre_hooks` too. It would make the code a bit nicer.